src/sql/workbench/parts/notebook/cellViews/output.component.html

```diff
@@ -5,7 +5,7 @@
  *--------------------------------------------------------------------------------------------*/
 -->
 <div style="overflow: hidden; width: 100%; height: 100%; display: flex; flex-flow: column">
-	<div style="flex: 0 0 auto; user-select: initial;">
-		<div #output ></div>
+	<div style="flex: 0 0 auto; user-select: none;">
+		<div #output class="output" ></div>
```

Comment:
>output [](start = 22, length = 6)

output is pretty generic - I'd suggest giving it a more unique name. I would suggest notebook-output normally but that's taken by the output area...so maybe consider renmaing that one to notebook-output-area and this to notebook-output

Reply:
In fact why even add the class? Just use the ID for the CSS

---
In reply to: [274515156](https://github.com/Microsoft/azuredatastudio/pull/4981#discussion_r274515156) [](ancestors = 274515156)